dask/delayed.py

```diff
@@ -6,8 +6,9 @@
 import operator
 import uuid
 
-from toolz import merge, unique, curry, first
+from toolz import merge, unique, curry, first, assoc
 
+from .async import get_sync
```

Comment:
I've gone through with flake8 separately.  I'd like to do this as a different PR